shared/src/commonMain/kotlin/co/nimblehq/mark/kmmic/data/service/auth/AuthService.kt

```diff
@@ -0,0 +1,30 @@
+package co.nimblehq.mark.kmmic.data.service.auth
+
+import co.nimblehq.mark.kmmic.data.service.api.ApiService
+import co.nimblehq.mark.kmmic.data.service.api.extension.path
+import co.nimblehq.mark.kmmic.data.service.auth.model.AuthLoginParams
+import co.nimblehq.mark.kmmic.data.service.auth.model.AuthTokenApiModel
+import io.ktor.client.request.HttpRequestBuilder
+import io.ktor.client.request.setBody
+import io.ktor.http.HttpMethod
+import kotlinx.coroutines.flow.Flow
+import org.koin.core.component.KoinComponent
+import org.koin.core.component.inject
+
+internal interface AuthService {
+    fun login(params: AuthLoginParams): Flow<AuthTokenApiModel>
+}
+
+internal class AuthServiceImpl: AuthService, KoinComponent {
+    private val apiService: ApiService by inject()
```

Comment:
Same issue here and in `LoginUseCaseImpl`.

Reply:
@AVI5HEK you can check this [file](https://github.com/markgravity/kmm-ic/blob/3225037d11388c90ad505fdf1de85c0ac1d7d6f2/shared/src/commonTest/kotlin/co/nimblehq/mark/kmmic/domain/usecase/GetCurrentUserUseCaseTest.kt) to see how I write test 🙏